examples/example_tf_transform/main.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018 Spotify AB.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+import os
+import shutil
+import tempfile
+import time
+
+import taxi
+import tensorflow as tf
+import tensorflow_transform as tft
+from examples.examples_utils import get_taxi_data_dir
+from spotify_tensorflow.tfx.tft import TFTransform
+
+
+class Preprocess(TFTransform):
```

Comment:
Does the example need to extend this class? I thought the script here could just define the `preprocessing_fn` and pass that into a call to `tftransform` along with the other args?

Reply:
@andrewsmartin forgot to push my last commit before commenting on the pr. I moved it out.  No, we don't need to extend this class. The script will be more like the one as https://github.com/spotify/spotify-tensorflow/blob/keshi/tft/examples/example_tf_transform/main.py

Reply:
Awesome! LGTM

Reply:
I didn't one more small tweak. 
I made `run`  in TFTransform a class member, and it'll be invoked as `TFTransform(preprocessing_fn=preprocessing_fn).run(args=args)`  
https://github.com/spotify/spotify-tensorflow/blob/keshi/tft/examples/example_tf_transform/main.py#L78

I feel it reads better.